test/unit/models/task.test.js

```diff
@@ -0,0 +1,141 @@
+const chai = require('chai')
+const { expect } = chai
+const tasks = require('../../../models/tasks')
+const sinon = require('sinon')
+const addUser = require('../../utils/addUser')
+const userData = require('../../fixtures/user/user')()
+const appOwner = userData[3]
+const otherUser = userData[0]
+
+describe('tasks', function () {
+  before(async function () {
+    await addUser(appOwner)
+    await addUser(otherUser)
+  })
+
+  afterEach(function () {
+    sinon.restore()
+  })
```

Comment:
We aren't using sinon anywhere, why do we need this then?
We should be using `cleanDb` instead, in afterEach 🤔 

Can you please explain what's going on here? I'm a bit confused